templates/reservation/_confirmNotification.html.twig

```diff
@@ -0,0 +1,32 @@
+{% apply inline_css(
+    source('https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css'),
+    source('https://use.fontawesome.com/releases/v5.7.0/css/all.css'),
+    source('https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.3.1/css/bootstrap.min.css'),
+    source('https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.7.5/css/mdb.min.css')
+) %}
```

Comment:
`Unable to find template "https:/stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" (looked into: /Users/senzowayne/PhpstormProjects/resavo/templates, /Users/senzowayne/PhpstormProjects/resavo/vendor/symfony/twig-bridge/Resources/views/Form).`

Reply:
Je tentais d'insérer les sources css... visiblement, ça n'est pas la bonne façon de faire.

Reply:
Ouai j’ai déjà vu pas mal de template de mailings et les gens font du style Css dans le template directement je crois que c’est une contrainte du templating pour le mail